config/ansible-lint.yml

```diff
@@ -8,3 +8,36 @@ skip_list:
   - 'no-changed-when'
   # Hide "Role name 'xxx_yyy' does not match ``^[a-z][a-z0-9_]+$`` pattern"
   - 'role-name'
+  ## The following Ansible lint rules are the ones that are currently
+  ## part of the main lint checks that are not adresses in this repo
+  ## The following skip rules represent 3143 failure(s), 12 warning(s) on 852 files.
```

Comment:
interesting, 
I think something stopped working with the GH Action at some point in the past, and the linting didn't run for quite a while now. 🤔 

Some of the rules should _not_ be skipped, like those for instance:
- command-instead-of-shell
- jinja[spacing]
- yaml[new-line-at-end-of-file]
- risky-shell-pip
- name[missing]

I'll create a ticket to track it when this PR is merged

Reply:
Yeah, this will be a huge fix, I wanted to address it in different PRs, the idea is having the job green, remove a rule with the corresponding fixes and then merge it step by step as the changes will be huge while keeping the job green.

Reply:
> I wanted to address it in different PRs

yes, 100% agree with that, that's the way I would have done it as well

Reply:
https://github.com/openshift-psap/topsail/issues/10